tests/Unit/Models/PageTest.php

```diff
@@ -77,4 +78,42 @@ public function page_url_will_adapt_to_changing_parent(): void
         $child->refresh();
         $this->assertSame($child->url, '/stuff/web-design');
     }
+
+    /** @test */
+    public function it_may_have_translations(): void
+    {
+        $page = Page::factory()->create();
+        $this->assertInstanceOf(Collection::class, $page->translations);
+    }
+
+    /** @test */
+    public function you_can_add_a_translation(): void
+    {
+        $pageEn = Page::factory()->create(['language' => 'en']);
+        $pageNl = Page::factory()->create(['language' => 'nl']);
+
+        $pageEn->translations()->attach($pageNl->id);
+
+        $pageEn->refresh();
+        $this->assertCount(1, $pageEn->translations);
+        $this->assertInstanceOf(Page::class, $pageEn->translations->first());
```

Comment:
Ik zou hier checken of de juiste translation eruit komt. Volgens mij kan je Eloquent models op deze manier gebruiken en anders hebben ze een compare functie.
```
        $this->assertEquals($pageNl, $pageEn->translations->first());
```

Reply:
Goeie inderdaad, ik voeg dit toe. :thumbsup: